### PR TITLE
Registration script

### PR DESCRIPTION
**Pull request by gal432**

### CODE REVIEW

File: lmsweb/registartion.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    def _send_user_email_registration(self, user: UserToCreate) -> None:
+        response = None
+        text = self._build_user_text(user)
+        url = F'https://api.eu.mailgun.net/v3/{config.MAILGUN_DOMAIN}/messages'
+        try:
+            response = self._session.post(
+                url=url,
+                data={
+                    'from': F'no-reply@{config.MAILGUN_DOMAIN}',
```
Comment (yammesicka) [inline lmsweb/registartion.py]: fstring with `f` instead of `F` for consistency?

File: lmsweb/registartion.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    def to_dict(self):
+        return self._asdict()
+
+    @classmethod
+    def get_fields(cls):
+        return cls._fields
+
+
+class UserRegistrationCreator(object):
```
Comment (yammesicka) [inline lmsweb/registartion.py]: Without `object`

File: lmsweb/registartion.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        if not os.path.exists(file_path):
+            raise ValueError
+
+        with open(file_path, 'r') as file_reader:
+            csv_records = csv.DictReader(file_reader)
+
+        users = []
+        for record in csv_records:
+            if 'password' not in record:
+                record['password'] = os.urandom(10).hex()
```
Comment (yammesicka) [inline lmsweb/registartion.py]: Const & a bit longer (12)? 'cause it's an hash.
Even better:

```
import string
import random

def generate_pass(length)
    ''.join(random.choices(string.printables.strip(), k=length))
````

File: lmsweb/registartion.py (hunk 2 above)
Comment (gal432) [reply lmsweb/registartion.py]: Such an python2 am I?

File: lmsweb/registartion.py (hunk 3 above)
Comment (gal432) [reply lmsweb/registartion.py]: Done

**Pull request merged**